develop/plone/searching_and_indexing/catalog.rst

```diff
@@ -93,17 +93,40 @@ Besides, the main portal catalog, the site contains other catalogs.
 Manually indexing object to a catalog
 -------------------------------------
 
-The default content object.reindexObject() is defined in
-`CMFCatalogAware <http://svn.zope.org/Products.CMFCore/trunk/Products/CMFCore/CMFCatalogAware.py?rev=102742&view=auto>`_
-and will update the object data to portal_catalog.
+The default content ``obj.reindexObject()`` is defined in
+`CMFCatalogAware <https://github.com/zopefoundation/Products.CMFCore/blob/2.2/Products/CMFCore/CMFCatalogAware.py#L78-L88>`_
+and will update the object data to ``portal_catalog``.
+
+If you don't need to reindex all the indexes you can speed up quite a bit by being more selective:
+
+.. code-block:: python
+
+    obj.reindexObject(idxs=['some_index'])
+
+You can use the ``portal_catalog`` directly if you need to update more than one object:
```

Comment:
I'm confused by this... in the code snippet you're still updating just the one `obj`. I guess I've never used `catalog_object`... need perhaps a bit more of an explanation or link to an explanation if there is one elsewhere?

Reply:
Question is, why would I use ``catalog.catalog_object(obj, idxs=['some_index'])`` for multiple objects? I do not get the advantage.

Reply:
I asked here: https://community.plone.org/t/best-practices-on-reindexing-the-catalog/4157/13?u=tkimnguyen

Reply:
@jensens `reindexObject()` call internally `catalog_object()`: we're saving looking up for the `portal_catalog` on every object.

Reply:
I must be thick, because I don't see where there are multiple lookups of portal_catalog. I do see where there is other logic being called repeatedly.

Reply:
https://github.com/zopefoundation/Products.CMFCore/blob/2.2/Products/CMFCore/CMFCatalogAware.py#L86